boat-scaffold/src/main/java/com/backbase/oss/codegen/angular/BoatAngularGenerator.java

```diff
@@ -221,6 +229,15 @@ private void applyAngularVersion(SemVer angularVersion) {
         }
     }
 
+    private void applyAngularVersion(SemVer angularVersion) {
```

Comment:
Related to my other 2 comments - if the conditional is removed and the minimum version check is added to the other overload, I don't think this overload is needed anymore.